uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/data/entitlement-data.js

```diff
@@ -66,54 +66,57 @@ define([
     breakdown: function () {
       return mockData.breakdown_data;
     }
-  }
+  };
 
   /**
    * Generates an entitlement for each absence type in each period
    *
    * @param {Boolean} withRemainder if the remainder data should be attached
    * @return {Array}
    */
-  function generateEntitlements(withRemainder) {
-    var i = 1, values = [];
+  function generateEntitlements (withRemainder) {
+    var i = 1;
```

Comment:
`i` is used for iterator, it's a common coding convention. Also there was a discussion related to this and we all agreed that `i` is fine.